### DISCUSSION TITLE
是否支持job或stage的状态监控API

### DISCUSSION THREAD

**Discussion by CCweixiao** [apache/kyuubi#421] [Q&A]
我们现在起了两套STS 作为高可用，但是遇到一些问题，有些用户提交了很长、很复杂的SQL，有时候执行很漫长，有时候跟卡住一样，job中的stage几个小时都没有进度，造成后续的任务无法提交，如果kyuubi支持job进度的监控，我们针对性杀掉这些异常的SQL提交，我们也尝试过用spark的rest API，但不理想

![image](https://user-images.githubusercontent.com/27730211/110748036-2ac35f00-827a-11eb-8049-3b91d340e226.png)

**Comment by yaooqinn** [comment]
How about make your question clear？Do you encounter any problem with Kyuubi?
It's better to follow the instructions here  https://kyuubi.readthedocs.io/en/latest/community/contributions.html and creating an issue w.r.t the issue template.

If you just want to ask questions for usages, please join this channel https://gitter.im/kyuubi-on-spark/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge

Bests and Enjoy with Kyuubi.

Thanks
